.github/workflows/cwin.yml

```diff
@@ -0,0 +1,32 @@
+name: cwin
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
The on: field needs to look like the other GitHub actions:

on:
  workflow_dispatch:
  push:
  pull_request:
    branches: [ master ]
    paths-ignore:
    - '.github/CODEOWNERS'
    - 'doc/**'
    - 'release_docs/**'
    - 'COPYING'
    - '**.md'